Content.Shared/_RMC14/Xenonids/Fruit/SharedXenoFruitSystem.cs

```diff
@@ -384,7 +384,7 @@ private void OnXenoFruitPlantAction(Entity<XenoFruitPlanterComponent> xeno, ref
             // Increase growth time if on hardy weeds
             var weeds = _xenoWeeds.GetWeedsOnFloor(coordinates, false);
             if (TryComp(weeds, out XenoWeedsComponent? weedsComp))
-                fruit.GrowTime = fruit.GrowTime / weedsComp.FruitGrowthMultiplier;
+                fruit.GrowTime = fruit.GrowTime * weedsComp.FruitGrowthMultiplier;
```

Comment:
*= even

Reply:
Problem is, GrowTime is a TimeSpan, and iirc I tried *= (or rather /=), but it didn't have that operator overloaded

Reply:
Never mind, apparently that works